rest-service/manager_rest/constants.py

```diff
@@ -14,10 +14,20 @@
 #  * limitations under the License.
 
 from collections import namedtuple
-from enum import Enum
 from typing import List, Union, Tuple
 
 
+try:
+    from enum import StrEnum
```

Comment:
It might make more sense to just support py3.11 and drop 3.10?

Reply:
That's not gonna be as direct... we'd have to build it first, etc.

Reply:
Fair enough, green.